PsesHeader header
float32 accelerometer_x
float32 accelerometer_y
float32 accelerometer_z
float32 gyroscope_x
float32 gyroscope_y
float32 gyroscope_z
float32 hall_sensor_dt
float32 hall_sensor_dt_full
uint32 hall_sensor_count
float32 range_sensor_front
float32 range_sensor_left
float32 range_sensor_right
float32 system_battery_voltage
float32 motor_battery_voltage

